pkg/crc/machine/stop.go

```diff
@@ -15,6 +18,27 @@ func (client *client) Stop() (state.State, error) {
 		return state.Error, errors.Wrap(err, "Cannot load machine")
 	}
 
+	// L#21-37 should be removed after https://bugzilla.redhat.com/show_bug.cgi?id=1965992
```

Comment:
Maybe extract this block in a function?

Reply:
Also maybe we need to do this only for 4.8 bundle. Add `if strings.HasPrefix(crcBundleMetadata.GetOpenshiftVersion(), "4.8.") {` ?

Reply:
Maybe better as a helper in the `bundle` package then?
```
if crcBundleMetadata.CheckOpenShiftMajorVersion("4.8") {
...
}
```

Reply:
> Also maybe we need to do this only for 4.8 bundle. Add `if strings.HasPrefix(crcBundleMetadata.GetOpenshiftVersion(), "4.8.") {` ?

`stop` doesn't have any idea about which bundle used to start the cluster so may be use what version of `oc` client on the host which is always up to date with bundle?